src/transformers/models/plbart/modeling_plbart.py

```diff
@@ -1408,10 +1409,11 @@ def __init__(self, config: PLBartConfig, **kwargs):
     @add_start_docstrings_to_model_forward(PLBART_INPUTS_DOCSTRING)
     @add_code_sample_docstrings(
         processor_class=_TOKENIZER_FOR_DOC,
-        checkpoint=_CHECKPOINT_FOR_DOC,
+        checkpoint="hf-internal-testing/tiny-plbart",
```

Comment:
nit: I would still prefer to use `_CHECKPOINT_FOR_SEQUENCE_CLASSIFICATION`, but feel free to merge without this change.